### ISSUE TITLE
Allow manual input with verification flag?

### ISSUE THREAD

**Issue by baudtack** [porybox/porybox#796]
I'd love to use porybox but I don't have cfw and battle videos are way too tedious to use.

Has any thought been given to providing a manual input method through a web form? I imagine this would also trip a flag that shows anyone who looks at the mon that this was manual input and not uploaded by a file and so not verified. 

Just thinking it might allow users (i.e. me) to use porybox without having to risk hacking their 3ds.

**Comment by SnowPhoenix** [comment]
I'm not sure I understand the advantages of this over, say, a spreadsheet. The idea of Porybox is that tracks all the Pokémon's info for you from the files without having to manually input it.

If you really wanted to, this could also be done at present by entering the Pokémon's info in PKHeX, saving it as a pk7, and then uploading the resulting pk7 from there. I wouldn't recommend it, as you'd still have some inaccuracies in data that can't be viewed from in-game summaries (which would limit the usefulness of hack checks, and clone checks would be useless), but it's at least possible.

**Comment by Raia** [comment]
>I'm not sure I understand the advantages of this over, say, a spreadsheet. The idea of Porybox is that tracks all the Pokémon's info for you from the files without having to manually input it.

I agree. I think a manual input falls outside of the scope of what Porybox is designed for.
